internal/painter/gl/draw.go

```diff
@@ -29,6 +29,64 @@ func (p *painter) defineVertexArray(prog Program, name string, size, stride, off
 func (p *painter) drawCircle(circle *canvas.Circle, pos fyne.Position, frame fyne.Size) {
 	p.drawTextureWithDetails(circle, p.newGlCircleTexture, pos, circle.Size(), frame, canvas.ImageFillStretch,
 		1.0, paint.VectorPad(circle))
+
+	radius := circle.Size().Width / 2
+	if circle.Size().Height < circle.Size().Width {
+		radius = circle.Size().Height / 2
+	}
```

Comment:
```suggestion
	size := circe.Size()
	radius := size.Width / 2
	if size.Height < size.Width {
		radius = size.Height / 2
	}
```

Reply:
Sorry for the misspelling of circle there.

Reply:
No problem :)